plaidml/edsl/__init__.py

```diff
@@ -600,6 +600,40 @@ def TensorIndexes(count):
     return [TensorIndex() for i in range(count)]
 
 
+def Placeholder(*args, **kwargs):
+    """Creates a placeholder tensor, specified either by a combination of data
+       type and dimensions or by passing in a LogicalShape.
```

Comment:
Can you add tests that use all the combinations possible for this function?

Reply:
Done